File: src/main/java/org/library/BorrowTransaction.java
package org.library;

public class BorrowTransaction {
    private final String bookTitle;
    private final String borrower;
    private final String borrowDate;

    public BorrowTransaction(String title, String borrowerName, String date){
        bookTitle = title;
        borrower = borrowerName;
        borrowDate = date;
    }

    public String getBookTitle(){
        return bookTitle;
    }

    public String getBorrower(){
        return borrower;
    }

    public String getBorrowDate(){
        return borrowDate;
    }
}

File: main/BorrowTransaction.js
export default class BorrowTransaction {
  constructor(title, borrowerName, date) {
    this.bookTitle = title;
    this.borrower = borrowerName;
    this.borrowDate = date;
  }

  getBookTitle() {
    return this.bookTitle;
  }

  getBorrower() {
    return this.borrower;
  }

  getBorrowDate() {
    return this.borrowDate;
  }
}
